### PR TITLE
Additional tests for find_template

### PR DESCRIPTION
**Pull request by kuter**
- added tests 
- removed template_loader workaround
- changed handle function

### CODE REVIEW

File: django_extensions/management/commands/find_template.py [hunk 1]
```diff
@@ -7,33 +7,16 @@
...
     help = "Finds the location of the given template by resolving its path"
     args = "[template_path]"
     label = 'template path'
 
     @signalcommand
     def handle_label(self, template_path, **options):
-        path = get_template_path(template_path)
-        if path is None:
+        try:
+            template = loader.get_template(template_path).template
```
Comment (trbs) [inline django_extensions/management/commands/find_template.py]: Are we sure this does not break anything ?

File: django_extensions/management/commands/find_template.py [hunk 2]
```diff
@@ -7,33 +7,16 @@
...
+        except TemplateDoesNotExist:
             sys.stderr.write("No template found\n")
-            sys.exit(1)
         else:
-            print(path)
+            print(template.name)
```
Comment (trbs) [inline django_extensions/management/commands/find_template.py]: Maybe we can change this to use `self.stdout.write(self.style.SUCCESS(...))` at the same time ?

**Comment by kuter** [comment]
@trbs take a look here, I've made changes based on your comments. If its ok .. then please merge that.

**Comment by trbs** [comment]
Thanks !

**Pull request merged**